TWLight/applications/signals.py

```diff
@@ -17,4 +23,23 @@ def under_discussion(sender, comment, request, **kwargs):
 
     if application_object.status == Application.PENDING and application_object.editor.user.username != request.user.username:
         application_object.status = Application.QUESTION
-        application_object.save()
\ No newline at end of file
+        application_object.save()
+
+
+@receiver(no_more_accounts)
+def set_partner_status(sender, **kwargs):
```

Comment:
Final thought: Is there a reason this is a separate and manually triggered signal, rather than having this be part of the `post_save` trigger for Applications? My initial thought with signals was that was where it would happen so that _however_ we set an application approved, it would always trigger, rather than requiring a signal be sent manually.

Reply:
I think signals.py is a more appropriate place to do this since we are trying to make some changes from `applications` that would implicate the `resources` application, I could be wrong though. I also like the manual trigger of signal for readability purposes.

Reply:
>we are trying to make some changes from applications that would implicate the resources application

That's not inherently an issue - in the current `post_save` we're creating Authorization objects, for example: https://github.com/WikipediaLibrary/TWLight/blob/master/TWLight/applications/models.py#L356

@jsnshrmn do you have thoughts on this? The core issue being triggering a manual signal to set partners to waitlist when approving the final available application vs. having the logic in the application `post_save` signal.

Reply:
I agree with @Samwalton9, but I've also let this sit for quite some time. I'm inclined to go ahead and approve this since the code is working.